blocks/copd-causes/copd-causes.css

```diff
@@ -0,0 +1,107 @@
+.block.copd-causes {
+    background: unset;
+    padding: 14px;
+    margin: auto;
+    display: grid;
+    gap: 0;
+    align-items: end;
+    justify-content: center;
+    max-width: 1000px;
+    grid-auto-rows: minmax(110px, auto);
+}
+
+.block.copd-causes > div {
+    display: grid;
+    grid-template-columns: 68px 1fr;
+    grid-template-rows: repeat(2, 1fr);
```

Comment:
This is making the first item look higher because it has a longer title.
I think it should be:
grid-template-rows: auto;